addons/web_editor/static/src/js/editor/odoo-editor/src/utils/utils.js

```diff
@@ -1751,6 +1752,57 @@ export function insertListAfter(afterNode, mode, content = []) {
     return list;
 }
 
+export function toggleList(node, mode, offset = 0) {
+    let pnode = node.closest('ul, ol');
+    if (!pnode) return;
+    const listMode = getListMode(pnode) + mode;
+    if (['OLCL', 'ULCL'].includes(listMode)) {
+        pnode.classList.add('o_checklist');
+        for (let li = pnode.firstElementChild; li !== null; li = li.nextElementSibling) {
+            if (li.style.listStyle != 'none') {
+                li.style.listStyle = null;
+                if (!li.style.all) li.removeAttribute('style');
```

Comment:
Why check `li.style.all` but no other styles? Are other styles not relevant?
Also,

```suggestion
                if (!li.style.all) {
                    li.removeAttribute('style');
                }
```

Reply:
> Why check li.style.all but no other styles? Are other styles not relevant?

Well I just moved it out of `HTMLLIElement.prototype.oToggleList` to `toggleList`. It's there from the early days of editor. But a bigger question is why do we wanna remove style attribute? let's say we have `<ul><li style="color: red">abc</li></ul>` (this is not a case right now but may be case after [task](https://www.odoo.com/web#id=3281223&cids=2&menu_id=4720&action=333&active_id=1695&model=project.task&view_type=form)) and now I change it to checked list the red color is gone. Do we want that?